repose-aggregator/components/services/atom-feed-service/atom-feed-service-impl/src/main/scala/org/openrepose/nodeservice/atomfeed/impl/actors/FeedReader.scala

```diff
@@ -92,7 +93,16 @@ class FeedReader(feedUri: String,
       MDC.put(TracingKey.TRACING_KEY, requestId)
 
       try {
-        val entryStream = AtomEntryStreamBuilder.build(feedUrl, AuthenticationRequestContextImpl(reposeVersion, requestId), authenticatedRequestFactory)
+        def getStream(): Stream[Entry] = {
+          AtomEntryStreamBuilder.build(feedUrl, AuthenticationRequestContextImpl(reposeVersion, requestId), authenticatedRequestFactory)
+        }
```

Comment:
You can also drop the empty parameter list here (i.e., `getStream()` can become `getStream()`). IntelliJ will whine if you leave the empty parameter list there.

If you wanted, considering how simple this function is, you could even drop the return type and the curly braces. That comes down to personal preference, though.

Just for your reference:
http://docs.scala-lang.org/style/declarations.html#methods
http://docs.scala-lang.org/style/declarations.html#body